readinglist/resource.py

```diff
@@ -216,10 +220,12 @@ def _extract_filters(self, queryparams):
             param = param.strip()
             value = native_value(value)
 
-            if param == '_sort':
+            # Ignore specific fields
+            if param.startswith('_') and param not in ('_since', '_to'):
```

Comment:
Ok I get it! Maybe in 6 months we will like to read it in the comment, no ? ``# Ignore specific fields like _sort _limit or _token``